sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -509,7 +509,7 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
           assert(
             intercept[AnalysisException] {
               sparkSession.catalog.createExternalTable("createdJsonTable", jsonFilePath.toString)
-            }.getMessage.contains("Table default.createdJsonTable already exists."),
+            }.getMessage.contains("Table `default`.`createdJsonTable` already exists."),
```

Comment:
Actually, I have a question about `unquotedString`. When should we issue an `unquotedString`? Unless we explicitly specify it, [we always use `quotedString`](https://github.com/apache/spark/blob/f036dd7ce727b40877337da66d687214786c4f14/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/identifiers.scala#L40) for `TableIdentifier`. For `FunctionIdentifier`, [we use an `unquotedString`](https://github.com/apache/spark/blob/f036dd7ce727b40877337da66d687214786c4f14/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/identifiers.scala#L75).

Now, very few places are issuing `unquotedString` for table identifiers. Should correct them to make all the messages consistent?